### ISSUE TITLE
NULL attribute value to delete attributes

### ISSUE THREAD

**Issue by kzangeli** [Fiware/context.Orion-LD#667]
For the following two requests:
```
- PATCH /entities/{EID}/attrs
- POST /entities/{EID}/attrs
```
Any of the attributes having a NULL value (instead of an object) must be deleted.
Those removed attributes must also be marked as DELETED in TRoE.

## Background
From the spec (1.3.1) about the usage of NULL to delete attributes:

5.2 Data Types
  None of the members described admit a null value, except when they are used in the context of an update operation (see
  clause 5.5.8) and implementations shall raise an error of type BadRequestData if a null value is encountered.

5.4 NGSI-LD Fragments
  A member (following the same data representation and nesting structure) with value equal to null for each
  member to be removed from the target NGSI-LD element.

5.6.4 Partial Attribute update
  The value of a Property or the object of a Relationship shall not be set to null in order to delete it as
  this would result in an incomplete Attribute instance.

5.6.7 Batch Entity Creation
  If the input Array is empty or contains a null value in any of its items an error of type BadRequestData shall be raised.

5.6.8 Batch Entity Creation or Update (Upsert)
  If the input Array is empty or contains a null value in any of its items, an error of type BadRequestData shall be raised.

5.6.9 Batch Entity Update
  If the input Array is empty or contains a null value in any of its items, an error of type BadRequestData shall be raised.

5.6.10 Batch Entity Delete
  If the input Array is empty or contains a null value in any of its items, an error of type BadRequestData shall be raised.

NGSi-LD	requests that use Entity Fragment:
```
- Update Entity Attributes    (PATCH /entities/{EID}/attrs)                 Attrs set to NULL are to be removed
- Append Entity Attributes    (POST /entities/{EID}/attrs)    	      	    Attrs set to NULL are to be removed
- Partial Attribute update    (PATCH /entities/{EID}/attrs/{attrName})	    NULL is not	valid in this request
```
======================================================================================

**So, in short, for the following two requests only:**
```
- PATCH /entities/{EID}/attrs
- POST /entities/{EID}/attrs
```
**Any of the attributes having a NULL value (instead of an object) must be removed**

**Comment by kzangeli** [comment]
I proposed to ETSI ISG CIM to drop this feature and that's what we'll do.
Issue to be closed.
